changelog.md

```diff
@@ -18,12 +18,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Component edition:
   * Allowed to have a predefined list of values for Array attribute.
 * Add new environment variable `KEEP_CYPRESS_ATTRIBUTE` to keep all cypress attribute in html.
+* Set the position of the component according to the position of the mouse when dropped on `#root`.
 
 ### Changed
 
 * Update e2e test plugin to terrator-plugin.
 * Internal optimization of plugin usage.
 * New display for library list & component grid.
+* Update plugin core version in `package.json`.
```

Comment:
Not sure if must be done on this PR

Reply:
I agree

Reply:
But where can we do it? this PR depends on the new version of the plugin-core